benchmark/Microsoft.IdentityModel.Benchmarks/ValidateTokenAsyncWithVPTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.IdentityModel.Benchmarks
+{
+    // dotnet run -c release -f net8.0 --filter Microsoft.IdentityModel.Benchmarks.ValidateTokenAsyncWithVPTests*
+
+    public class ValidateTokenAsyncWithVPTests
```

Comment:
nit: Per [naming guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms) abbreviations shouldn't be used often unless they are well known. In this case, it wasn't obvious what `VP` stood for. So I would maybe spell out `ValidationParameters` or describe in some other way the difference between new vs old behaviors. It does lengthen the names but I think it makes it clearer.

Reply:
Makes sense, updated. Thanks Peter!